src/System.ValueTuple/src/System/ValueTuple/ValueTuple.cs

```diff
@@ -5,7 +5,6 @@
 using System.Collections;
 using System.Collections.Generic;
 using System.Diagnostics;
-using System.Numerics.Hashing;
 using System.Runtime.InteropServices;
 
 namespace System
```

Comment:
Does F# depend on the behavior of this, too? (see https://github.com/dotnet/coreclr/blob/master/src/mscorlib/shared/System/Tuple.cs#L12)

Reply:
@jkotas , it looks like you added that comment. Can you please expand on what it's about and what might go wrong if we change it?

Reply:
F# depends on the exact hash code for Tuple only. It is ok to change ValueTuple.

Reply:
This is CoreFX legacy version of ValueTuple. It does not have access to HashCode. I think you should leave it as is.